meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
Are you sure? I put it into the section "Dynamic" on purpose, because that's where it belongs and was missing. See full file:
https://github.com/pohly/ostro-os/blob/meta-swupd3/meta-ostro/conf/distro/include/ostroproject-ci.inc